hudi-common/src/main/java/org/apache/hudi/metadata/BaseTableMetadata.java

```diff
@@ -73,8 +73,10 @@ public abstract class BaseTableMetadata implements HoodieTableMetadata {
 
   private static final Logger LOG = LogManager.getLogger(BaseTableMetadata.class);
 
-  public static final long MAX_MEMORY_SIZE_IN_BYTES = 1024 * 1024 * 1024;
-  public static final int BUFFER_SIZE = 10 * 1024 * 1024;
+  protected static final long MAX_MEMORY_SIZE_IN_BYTES = 1024 * 1024 * 1024;
+  // NOTE: Buffer-size is deliberately set pretty low, since MT internally is relying
+  //       on HFile (serving as persisted binary key-value mapping) to do caching
+  protected static final int BUFFER_SIZE = 10 * 1024; // 10Kb
```

Comment:
What we prefetch here isn't actually used past parsing the block header (magic, version, etc) so there's no point in fetching large slabs. The reason why this is not going to be used is b/c we use HFile reader which does its own caching of the blocks

Reply:
did you get a chance to run any benchmarks w/ this fix? 
even w. Hfile, if the inputStream size is set to 10Kb, while original hfile block size is 64Kb, we might make more remote calls just to fetch 1 hfile block right? instead of just 1.

Reply:
This has nothing to do w/ HFile block --  this setting simply controls the buffer-size for `BufferedFSInputStream` which is only used to parse headers/footers of the delta log-file and is completely separate from HFile reader we're actually using to read MT